src/utils/testHelpers.js

```diff
@@ -0,0 +1,18 @@
+/* eslint-disable import/no-extraneous-dependencies */
```

Comment:
- [ ] I request that you re-enable the linters and fix them instead. They help to create good quality code that is also well formatted.